tdrs-backend/tdpservice/users/api/login.py

```diff
@@ -395,6 +395,7 @@ class CypressLoginDotGovAuthenticationOverride(TokenAuthorizationOIDC):
 
     def post(self, request):
         """Create a session for the specified user, if they exist."""
+        logging.debug(f"{self.__class__.__name__}: {request} ; {request.data}")
```

Comment:
is the logging still needed or can it be removed?

Reply:
Not required of course, but I found it helpful and thought it could still be useful at debug level; is it too much?